src/main/java/flash/npcmod/entity/NpcEntity.java

```diff
@@ -13,9 +13,10 @@
 import flash.npcmod.core.quests.QuestInstance;
```

Comment:
I can only add a review comment here because git is great, but don't use client-only stuff in a class that's not client-only:
```
import flash.npcmod.client.gui.behavior.Action;
import flash.npcmod.client.gui.behavior.Behavior;
import flash.npcmod.client.gui.behavior.Trigger;
```

This'll 100% crash on a server, because the server doesn't know about these classes.